spring-integration-gcp/README.adoc

```diff
@@ -34,3 +36,74 @@ These are useful to process the message ID, in case of success, or the error if
 
 Refer to the https://spring.io/guides/gs/spring-cloud-gcp/[Spring getting started guide] to learn
 how to use the adapters.
+
+== Google Cloud Storage
```

Comment:
Tried something like `1include::/spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-storage.adoc[lines=33..-1]`, but it didn't work.
I think it might be because this README is processed differently from the reference document.
I don't know of another way to make this work..